src/components/Team/index.js

```diff
@@ -120,6 +120,7 @@ class Team extends Component {
           user={user}
           links={links}
           shortcutLinks={shortcutLinks}
+          toggled={true}
```

Comment:
```suggestion
          toggled
```

when a prop is a boolean, don't need to set a value for it. a `<Component toggled>` will have a prop `toggled` as `true`. So, in the end `<Component toggled>` and `<Component toggled={true}>` are the same thing

Reply:
Cool, thanks!